src/vm/interpreter.cpp

```diff
@@ -1431,8 +1431,8 @@ CorJitResult Interpreter::GenerateInterpreterStub(CEEInfo* comp,
         sl.X86EmitPopReg(kEBP);
         sl.X86EmitReturn(static_cast<WORD>(argState.callerArgStackSlots * sizeof(void*)));
 #elif defined(_AMD64_)
-        // EDX has "ilArgs" i.e., just the point where registers have been homed.
-        sl.X86EmitIndexLeaRSP(kEDX, static_cast<X86Reg>(kESP_Unsafe), 8);
+        // Pass "ilArgs", i.e. just the point where registers have been homed, as 2nd arg
+        sl.X86EmitIndexLeaRSP(kRSI, static_cast<X86Reg>(kESP_Unsafe), 8);
```

Comment:
@janvorli Thanks you for comment. I wasn't aware of Linux/Windows ABI difference on x64. I plan to revise PR as you suggested, but 2nd argument is passed via ``EDX`` (instead of ``RDX``)?